pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/migration/Physician.java

```diff
@@ -20,6 +20,9 @@ public final class Physician {
     @SerializedName("state")
     private String state;
 
+    @SerializedName("country")
+    private String country;
+
```

Comment:
What is the expected behavior if a client passes `null` for this field? Given there are a number of clients using this API, and they may not all be updated, this should be backwards compatible with any currently deployed sites.

Reply:
Actually we discussed the scenario, when an user manually enters the country name which is not present in country_address_info, if we consider the current state of user_medical_provider table that kind of data will be lost. So maybe it will be better if we revert the column type back to varchar and put country names directly without relations?

Reply:
This issue is also described in https://broadinstitute.atlassian.net/browse/DDP-8631

Reply:
I talked this over with @MocanaAtBroad- let's revert this back to just storing the bare string in the database, and not worrying about making sure it's a proper foreign reference.